Alignment/OfflineValidation/plugins/PrimaryVertexValidation.cc

```diff
@@ -3710,18 +3711,8 @@ void PrimaryVertexValidation::fillDescriptions(edm::ConfigurationDescriptions& d
   desc.add<edm::InputTag>("BeamSpotTag", edm::InputTag("offlineBeamSpot"));
 
   // track filtering
-
   edm::ParameterSetDescription psd0;
-  psd0.add<double>("maxNormalizedChi2", 5.0);
-  psd0.add<double>("minPt", 0.0);
-  psd0.add<std::string>("algorithm", "filter");
-  psd0.add<double>("maxEta", 5.0);
-  psd0.add<double>("maxD0Significance", 5.0);
-  psd0.add<double>("maxD0Error", 1.0);
-  psd0.add<double>("maxDzError", 1.0);
-  psd0.add<std::string>("trackQuality", "any");
-  psd0.add<int>("minPixelLayersWithHits", 2);
-  psd0.add<int>("minSiliconLayersWithHits", 5);
+  TrackFilterForPVFinding::fillPSetDescription(psd0);
```

Comment:
The parameters in the old version of this file do not all match those at https://github.com/cms-sw/cmssw/pull/33104/files#diff-5f0d748ca073d46b3b494a2323110fd87749fed3e0d6c72fcff85841d47b17adR51. Should you have the following here:

```
psd0.add<double>("maxNormalizedChi2", 5.0);
psd0.add<double>("maxEta", 5.0);
psd0.add<double>("maxD0Significance", 5.0);
```

Or is the change intended?

Reply:
these values are anyway overridden in the actual configurations:

https://github.com/cms-sw/cmssw/blob/e53c740bfd118fa874c0c095864809b41c84c58f/Alignment/OfflineValidation/python/TkAlAllInOneTool/primaryVertexValidationTemplates.py#L75-L79

so in the it does not matter.

Reply:
OK, thanks for the explanation.

Reply:
Now come to think of it, your question stimulated my curiosity. 
@perrotta is there a way to use only part of the `fillPSetDescription` of another module or is it considered good practice to overwrite the parts that can differ locally as suggested above in https://github.com/cms-sw/cmssw/pull/33104#discussion_r591898863 ?

Reply:
@mmusich I think that the most versatile way of assigning different parameters is exactly updating them in the explicit configuration, as you did. Configutability of those parameters is exactly intended for it.

On the other hand, I have no idea whether it is possible to overwrite a given parameter in the `ParameterSetDescription`: there is not a `modify` method, only `add`, and I never tested what happens when you `add` twice the same parameter with a different default value.

Reply:
> On the other hand, I have no idea whether it is possible to overwrite a given parameter in the `ParameterSetDescription`: there is not a `modify` method, only `add`, and I never tested what happens when you `add` twice the same parameter with a different default value.

By reading the code `ParameterSetDescription` should throw an exception in case of duplicate parameter names. One way to get different defaults in the generated `PSet`s would be to pass those values to `fillPSetDescription()` function via arguments.

Reply:
Thanks @makortel !